homeassistant/components/cmus/media_player.py

```diff
@@ -86,11 +86,38 @@ def __init__(self, server, password, port, name):
             self.cmus = remote.PyCmus()
             auto_name = "cmus-local"
         self._name = name or auto_name
+        self._server = server
+        self._password = password
+        self._port = port
         self.status = {}
 
+    def reconnect(self):
+        """Reconnect the CMUS device."""
+
+        try:
+            if self._server:
+                self.cmus = remote.PyCmus(
+                    server=self._server, password=self._password, port=self._port
+                )
+            else:
+                self.cmus = remote.PyCmus()
+        except exceptions.InvalidPassword:
+            _LOGGER.error("The provided password was rejected by cmus")
+            return False
```

Comment:
When an exception is thrown, this method returns a boolean which is not used. When no exception is thrown, this method doesn't return anything.

Reply:
Thanks.